buildlib/pr/main.yml

```diff
@@ -47,7 +47,7 @@ resources:
       options: $(DOCKER_OPT_ARGS) $(DOCKER_OPT_VOLUMES)
     - container: ubuntu2204
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/x86_64/ubuntu22.04/builder:mofed-5.7-0.2.3.0
-      options: $(DOCKER_OPT_ARGS) $(DOCKER_OPT_VOLUMES)
+      options: $(DOCKER_OPT_ARGS) $(DOCKER_OPT_VOLUMES) $(DOCKER_OPT_IB)
```

Comment:
where else is this image used? Maybe it's worth to define  separate image with postfix `_ib`, like it is done for `centos7_ib`